### PR TITLE
HADOOP-16334. Fix yetus-wrapper not working when HADOOP_YETUS_VERSION greater or equal than 0.9.0

### PR DESCRIPTION
**Pull request by jiwq**
[HADOOP-16334](https://issues.apache.org/jira/browse/HADOOP-16334)

### CODE REVIEW

File: dev-support/bin/yetus-wrapper [hunk 1]
```diff
@@ -77,11 +81,17 @@ HADOOP_YETUS_VERSION=${HADOOP_YETUS_VERSION:-0.8.0}
 BIN=$(yetus_abs "${BASH_SOURCE-$0}")
 BINDIR=$(dirname "${BIN}")
 
+## HADOOP_YETUS_VERSION >= 0.9.0 the tarball named with apache-yetus prefix
+if version_ge $HADOOP_YETUS_VERSION 0.9.0; then
```
Comment (hadoop-yetus) [inline dev-support/bin/yetus-wrapper]: shellcheck:15: note: Double quote to prevent globbing and word splitting. [SC2086]

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 903 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| 0 | shelldocs | 0 | Shelldocs was not available. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
||| _ trunk Compile Tests _ |
| +1 | shadedclient | 790 | branch has no errors when building and testing our client artifacts. |
||| _ Patch Compile Tests _ |
| -1 | shellcheck | 0 | The patch generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0) |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 747 | patch has no errors when building and testing our client artifacts. |
||| _ Other Tests _ |
| -1 | asflicense | 30 | The patch generated 17 ASF License warnings. |
| | | 9910 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-864/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/864 |
| Optional Tests | dupname asflicense shellcheck shelldocs |
| uname | Linux 58d2bae0e219 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 06fa4d2 |
| shellcheck | https://builds.apache.org/job/hadoop-multibranch/job/PR-864/1/artifact/out/diff-patch-shellcheck.txt |
| asflicense | https://builds.apache.org/job/hadoop-multibranch/job/PR-864/1/artifact/out/patch-asflicense-problems.txt |
| Max. process+thread count | 410 (vs. ulimit of 5500) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-864/1/console |
| versions | git=2.7.4 maven=3.3.9 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 46 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| 0 | shelldocs | 0 | Shelldocs was not available. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
||| _ trunk Compile Tests _ |
| +1 | shadedclient | 751 | branch has no errors when building and testing our client artifacts. |
||| _ Patch Compile Tests _ |
| +1 | shellcheck | 0 | There were no new shellcheck issues. |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 772 | patch has no errors when building and testing our client artifacts. |
||| _ Other Tests _ |
| -1 | asflicense | 35 | The patch generated 17 ASF License warnings. |
| | | 1725 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-864/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/864 |
| Optional Tests | dupname asflicense shellcheck shelldocs |
| uname | Linux 1861752cbada 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 7f2e87a |
| asflicense | https://builds.apache.org/job/hadoop-multibranch/job/PR-864/2/artifact/out/patch-asflicense-problems.txt |
| Max. process+thread count | 414 (vs. ulimit of 5500) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-864/2/console |
| versions | git=2.7.4 maven=3.3.9 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by jiwq** [comment]
Hi @aajisaka, can you take some time to help review?

**Comment by aajisaka** [comment]
LGTM, +1

**Comment by jiwq** [comment]
Thanks for @aajisaka reviewed and committed.

**Pull request closed without merging**